packages/ui-components/src/components/Menu/Menu.component.tsx

```diff
@@ -48,7 +50,7 @@ export const Menu = ({ children = null, variant = "normal", className = "", ...p
 					juno-menu 
 					juno-menu-${variant} 
 					${baseStyles} 
-					${variantStyles(variant)} 
+					${variant === "small" ? smallStyles : normalStyles} 
```

Comment:
is my github looks crazy or iti s formating issue?

Reply:
Not exactly sure what you mean? I mainly tried to get rid of an extra function for this simple thing. In fact, we could get rid of using string literals here and use the tw-classes straightaway imho.